README.md

```diff
@@ -103,14 +103,16 @@ function App() {
     console.log("error code:", event.error, "error messsage:", event.message);
   });
 
-  const handleStart = () => {
-    ExpoSpeechRecognitionModule.requestPermissionsAsync().then((result) => {
+  const handleStart = async () => {
+    try {
+      const result =
+        await ExpoSpeechRecognitionModule.requestPermissionsAsync();
       if (!result.granted) {
         console.warn("Permissions not granted", result);
         return;
       }
       // Start speech recognition
-      ExpoSpeechRecognitionModule.start({
+      await ExpoSpeechRecognitionModule.start({
```

Comment:
This function is synchronous so it shouldn't need to be awaited. Asynchronous behavior for this function is captured through the events (start,stop,error,speechstart,result etc)